docs/reference/cli.txt

```diff
@@ -52,6 +52,11 @@ equivalent:
    det s
    det slot list
 
+Commands that launch Determined tasks can take :ref:`configurations
```

Comment:
Not a huge fan of "configurations" -- I'd go with "configuration" or "configuration options".